ui/ducks/confirm-transaction/confirm-transaction.duck.js

```diff
@@ -308,3 +344,38 @@ export function clearConfirmTransaction() {
     type: CLEAR_CONFIRM_TRANSACTION,
   };
 }
+
+export const updateTxValueIfMaxEthSettled = (
+  dispatch,
+  state,
+  { gasFeeEstimates, gasEstimateType },
+) => {
+  const txId = state.confirmTransaction?.txData?.id;
+
+  if (
```

Comment:
I've noticed in `updateTxValueIfMaxEthSettled` function that the entire logic is wrapped inside an `if` block. Implementing an early return pattern here could enhance the code's readability and reduce unnecessary nesting. This practice is generally beneficial for maintaining cleaner, more readable code. Here's a suggested refactor:

Current Implementation:
```javascript
function foo() {
  if (condition) {
    // do the stuff
  }
}
```

Suggested Refactor:
```javascript
function foo() {
  if (!condition) {
    return;
  }
  // do the stuff
}
```

This approach brings the main logic of the function to a higher level of indentation, making it more immediately visible and understandable.

Reply:
This make sense Derek, I will keep this in mind next time